aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -61,7 +61,93 @@ Use the **Razor Class Library** project template.
+* Adds a NuGet package reference for the [Microsoft.AspNetCore.Components.Web](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.Web) package.
+
+Example:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>{TARGET FRAMEWORK}</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <SupportedPlatform Include="browser" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="{VERSION}" />
+  </ItemGroup>
+
+</Project>
+```
+
+In the preceding example:
+
+* The `{TARGET FRAMEWORK}` placeholder is the [Target Framework Moniker (TFM)](/dotnet/standard/frameworks).
+* The `{VERSION}` placeholder is the version of the [`Microsoft.AspNetCore.Components.Web`](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.Web) package.
+
+### Only support the Blazor Server hosting model
+
+Class libraries rarely only support [Blazor Server](xref:blazor/hosting-models#blazor-server) apps. If the class library requires [Blazor Server](xref:blazor/hosting-models#blazor-server)-specific features, such as access to <xref:Microsoft.AspNetCore.Components.Server.Circuits.CircuitHandler>s or <xref:Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage>, or uses ASP.NET Core-specific features, such as middleware, MVC controllers, or Razor Pages, use **one** of the following approaches:
+
+* Specify that the library supports pages and views when the project is created with the **Support pages and views** check box (Visual Studio) or the `-s|--support-pages-and-views` option with the `dotnet new` command:
+
+  ```dotnetcli
+  dotnet new razorclasslib -s true
+  ```
+
+* Only provide a framework reference to ASP.NET Core in the library's project file:
+
+  ```xml
+  <Project Sdk="Microsoft.NET.Sdk.Razor">
+
+    <ItemGroup>
+      <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    </ItemGroup>
+
+  </Project>
+  ```
+
+### Support multiple framework versions
+
+If the library must support features added to Blazor in the current release while also supporting one or more earlier releases, multi-target the library. Provide a semicolon-separated list of [Target Framework Monikers (TFMs)](/dotnet/standard/frameworks) in the `TargetFrameworks` MSBuild property:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFrameworks>{TARGET FRAMEWORKS}</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <SupportedPlatform Include="browser" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="{VERSION}" />
+  </ItemGroup>
+
+</Project>
+```
+
+In the preceding example:
+
+* The `{TARGET FRAMEWORKS}` placeholder represents the semicolon-separated [Target Framework Monikers (TFMs)](/dotnet/standard/frameworks) list, for example: `netcoreapp3.1;net5.0`.
```

Comment:
It feels like overkill to link to the TFMs doc again, since it's already done in this section.

```suggestion
* The `{TARGET FRAMEWORKS}` placeholder represents the semicolon-separated TFMs list. For example, `netcoreapp3.1;net5.0`.
```

Reply:
Readers land anywhere in a topic from search engines and cross-links ... but ok.

Reply:
I worry a bit about incomplete sentence structures for loc ... but again ... I defer to your *stylings*&trade; :smile:.

Reply:
Oh ... "in this section" ... I see what u mean. :+1: